x-pack/plugins/reporting/common/types/export_types/csv_v2.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import type { BaseParamsV2, BasePayloadV2 } from '../base';
+import type { BaseParamsV2, BasePayloadV2 } from '@kbn/generate-csv';
```

Comment:
we should keep this as it was before, or replace with
```suggestion
import type { BaseParamsV2, BasePayloadV2 } from '..';
```